apps/retrospective/models/sprint.go

```diff
@@ -146,6 +148,7 @@ func (sprint *Sprint) validateDateContinuity(baseQuery *gorm.DB, statuses []Spri
 
 // BeforeSave ...
 func (sprint *Sprint) BeforeSave(db *gorm.DB) (err error) {
+	sprint.SprintID = strings.TrimSpace(sprint.SprintID)
```

Comment:
Shouldn't we do the same for sprint title?

Reply:
done